pkg/extension/sumologicextension/README.md

```diff
@@ -67,16 +67,15 @@ and can be used as an authenticator for the
   - `max_interval` - maximum interval of backoff (default: `1m`)
   - `max_elapsed_time` - time after which registration fails definitely (default: `15m`)
 
-[credentials_help]: https://help.sumologic.com/Manage/Security/Access-Keys
+[credentials_help]: https://help.sumologic.com/Manage/Security/Installation_Tokens
 [fields_help]: https://help.sumologic.com/Manage/Fields
 
 ## Example Config
 
 ```yaml
 extensions:
   sumologic:
-    access_id: aaa
-    access_key: bbbbbbbbbbbbbbbbbbbbbb
+    install_token: U1VNT1RhSHd3eU9oTDhHSkxmZjdhRG5lbFYzODA3Q3ZodHRwczovL3ZtYW5kYWwtdm1hbmRhbC1tYWMtZXZlbnRzLnN1bW9sb2dpYy5uZXQ=
```

Comment:
Is that a live token? 🙃 Can this be made to look a bit more like a dummy placeholder?

Reply:
Of course it's not live, it's an example I stole from backend tests. I wanted to show how an actual token looks like. Can replace with a placeholder if you feel strongly about it.

Reply:
I'd prefer a placeholder like in other docs

Reply:
Ok, changed.